osh/cmd_eval.py

```diff
@@ -381,47 +381,46 @@ def _CheckStatus(self, status, cmd_st, node, blame_spid):
           #   ls *Z          # error.FailGlob
           #   sort <(ls /x)  # process sub failure
           desc = 'Command'
-          span_id = location.SpanForCommand(node)
+          blame_loc_ = location.LocForCommand(node)
```

Comment:
The name conflicts are a bit annoying, I guess the param could be `default_loc` to be more distinct